src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -750,6 +761,57 @@ void regclass_graph_Model(py::module m) {
                 :rtype: int
              )");
 
+    model.def(
+        "get_sink_index",
+        [](ov::Model& self, const ov::Output<ov::Node>& value) -> int64_t {
+            int64_t pos = -1;
+            if (ov::is_type<ov::op::v6::Assign>(value.get_node_shared_ptr())) {
+                auto sink = std::dynamic_pointer_cast<ov::op::Sink>(value.get_node_shared_ptr());
+                pos = find_sink_position(self.get_sinks(), sink);
```

Comment:
```suggestion
                return find_sink_position(self.get_sinks(), sink);
```
`pos` can be remvoed.